WTWSMS/common/landed_titles/WtWSMS_holy_orders.txt

```diff
@@ -286,4 +286,65 @@ d_wasilla_guardians = {
 	# Extra income due to donations, etc
 	monthly_income = 10
 
+}
+
+c_priscillianism_holy_order = {
+		color = { 0.5 0.7 0.5 }
+		
+		graphical_culture = occitangfx # Closest thing to Iberian GFX in the game
+		
+		capital = 198 # Avila
+		
+		title = "PRISCMASTER"
+		foa = "HOCHMEISTER_FOA"
+		
+		landless = yes
+		
+		holy_order = yes
+		
+		religion = priscillianism
+		
+		# culture = cantabrian
+		
+		primary = yes
+		
+		short_name = yes
+		
+		name_tier = DUKE
+		
+		monthly_income = -0.5 # Asceticism (this is supposed to reduce the income by half)
+		
+		mercenary_type = mamluks_composition
+		
+		male_names = { Priscillian Marcus Helpidus }
+		
+		strength_growth_per_century = 0.15
+		
+}
+	
+d_sethian_holy_order = {
+		color = { 0.123 0.123 0.123 }
+		
+		graphical_culture = hashshashingfx # Mystic warriors of the Middle East
+		
+		capital = 802 # Alexandria
+		
+		title = "GRANDMASTER"
+		foa = "GRANDMASTER_FOA"
+		
+		landless = yes
+		
+		holy_order = yes
+		
+		religion = sethian
+		
+		primary = yes
+		
+		short_name = yes
+		
+		monthly_income = 1.5
+		
+		mercenary_type = sacred_band
+		
+		strength_growth_per_century = 0.05
```

Comment:
The entire mercenaries file was crap and needed a total overhaul. I did that because it needed to be done, and I don't appreciate you demanding a return to the spectacular failure the file once was.